catroid/src/main/java/org/catrobat/catroid/content/bricks/SetBackgroundBrick.java

```diff
@@ -44,6 +45,12 @@ public SetBackgroundBrick() {
 	}
 
 	@Override
+	public View getView(Context context) {
```

Comment:
this is not necessary. please remove it -> see my other comment for more context.